app/views/shared/_navbar.html.erb

```diff
@@ -10,9 +10,9 @@
     <% if user_signed_in? %>
 
       <!-- Links when logged in -->
-      <%= link_to "Host", "#", class: "navbar-wagon-item navbar-wagon-link" %>
-      <%= link_to "Trips", "#", class: "navbar-wagon-item navbar-wagon-link" %>
-      <%= link_to "Messages", "#", class: "navbar-wagon-item navbar-wagon-link" %>
+      <%= link_to "Host Hello", "#", class: "navbar-wagon-item navbar-wagon-link" %>
+      <%= link_to "Trips Thing", "#", class: "navbar-wagon-item navbar-wagon-link" %>
```

Comment:
FILIPE... WHY? this is not descriptive, you need to work on your naming. Please change this to something meaningful.

Reply:
?